src/main/scala/ai/privado/languageEngine/python/processor/PythonProcessor.scala

```diff
@@ -51,6 +52,7 @@ object PythonProcessor {
           // Apply default overlays
           X2Cpg.applyDefaultOverlays(cpg)
           new ImportsPass(cpg).createAndApply()
+          new ImportResolverPass(cpg).createAndApply()
```

Comment:
This import resolver needs to be added for JavaScript too

Reply:
We need to add `io.joern.pysrc2cpg.DynamicTypeHintFullNamePass` before `PythonTypeRecoveryPass`

Reply:
For JavaScript we are calling `io.joern.jssrc2cpg.JsSrc2Cpg#postProcessingPasses` method. Which has mentioned pass.

Reply:
@hiteshbedre ah yes I see